sos/report/plugins/openstack_keystone.py

```diff
@@ -106,6 +112,10 @@ def postproc(self):
             "|".join(connection_keys),
             r"\1*********\6"
         )
+        self.apply_regex_sub(
+            r"(^\s*(%s)\s*)(.*)" % "|".join(client_secrets),
+            r"\1*********"
+        )
```

Comment:
Could we not add `OIDCClientSecret` to the existing list of keys, and make the existing regex just have the `=` optional? E.G regex in line 107 becomes `r"(^\s*(%s)\s*(=\s*)?)(.*)"`, that way we can expand the keys being obfuscated without adding a call for each key?